helm/is/is/Chart.yaml

```diff
@@ -15,5 +15,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for the deployment of WSO2 IS Deployment
-name: is-deployment
+name: is
 version: 1.0.0
```

Comment:
Shall we check if this version corresponds to something helm charts specific or it is some dumb value that we provide, if it's something that we provide, we may rename this to the upcoming repo release, WDYT?

Reply:
Added the relevant product version for above in 5908da3